capella_model_explorer/state.py

```diff
@@ -0,0 +1,30 @@
+# Copyright DB InfraGO AG and contributors
+# SPDX-License-Identifier: Apache-2.0
+
+from __future__ import annotations
+
+import time
+import typing as t
+
+import prometheus_client
+
+if t.TYPE_CHECKING:
+    import capellambse
+    import jinja2
+
+    from capella_model_explorer import reports
+
+jinja_env: jinja2.Environment
+
+
+idle_time_gauge = prometheus_client.Gauge(
+    "idle_time_minutes",
+    "Time in minutes since the last user interaction",
+)
```

Comment:
Note this rename (`idletime_minutes` => `idle_time_minutes`) is a breaking change for monitoring setups. We need to make sure to update that and keep it consistent everywhere after the merge. That's why I proposed to rename the test functions rather than the metric in my earlier comment.

Reply:
Yes, mentioned that in the commit message already and your message led to another todo item in the PR description. Thanks!